h2o-docs/src/product/data-science/algo-params/hglm.rst

```diff
@@ -0,0 +1,80 @@
+``hglm``
+--------
+
+- Available in: GLM
+- Hyperparameter: no
+
+Description
+~~~~~~~~~~~
+
+Generalized Linear Models (GLM) estimate regression models for outcomes following exponential distributions. Hierarchical GLM (HGLM) fits generalized linear models with random effects, where the random effect can come from a conjugate exponential-family distribution (for example, Gaussian). HGLM allows you to specify both fixed and random effects, which allows fitting correlated to random effects as well as random regression models. 
+
+HGLM produces estimates for fixed effects, random effects, variance components and their standard errors. It also produces diagnostics, such as variances and leverages. 
```

Comment:
Angela:  Can you replace the It also produces diagnostics, such as variances and leverages with

It also produces diagnostics such as variances, dispersion parameters.

I don't think I am providing calculating for leverages yet.